.github/workflows/release-process.yml

```diff
@@ -0,0 +1,102 @@
+name: kapp-controller release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  kapp-controller-release:
+    name: kapp-controller release
+    runs-on: ubuntu-latest
```

Comment:
nit - i'd be tempted to pin to `ubuntu-20.04` just so that nobody has to scramble in two-ish years when ubuntu-latest suddenly points to 22.04 and some weird breaking change happens..